src/main/java/com/ubc/cpsc304/hotelify/entity/Review.java

```diff
@@ -0,0 +1,27 @@
+package com.ubc.cpsc304.hotelify.entity;
+
+import javax.persistence.*;
+
+import lombok.Data;
+
+@Data
+@Entity
+public class Review {
+
+    @Id
+    @Column(nullable = false, updatable = false, unique = true)
+    private Long review_id;
```

Comment:
review id is auto generated by hibernate so you need to add the annotation `@GeneratedValue(strategy = GenerationType.IDENTITY)`